### ISSUE TITLE
ID Evidence API

### ISSUE THREAD

**Issue by OscarGodson** [docusign/docusign-esign-node-client#357]
It doesn't look like it but is there ID Evidence support in the Node client?

If it doesn't, does the Node client provide any hooks for hitting unsupported APIs without needing to build the raw requests from scratch? For example, in pseudo code, 

```js
const dsApiClient = new docusign.ApiClient()
dsApiClient.setBasePath(OUR_ENV_VAR)
dsApiClient.addDefaultHeader('Authorization', `Bearer ${token}`)
dsApiClient.addDefaultHeader('Content-Transfer-Encoding', 'base64')
// Here im able to call an unsupported API but with the existing Node Client patterns.
const requestApi = new docusign.apiRequest(dsApiClient, { endpoint: '/events/account/123/456/media/789' })
```

**Comment by Adrian-DS-Support** [comment]
Hi Oscar, 

Our Node Client SDK was created specifically to access our DocuSign API. If you are having problems with the DocuSign API please let us know the details. 

Regards
Adrian
Developer Support

**Comment by OscarGodson** [comment]
Yes, I'm referring to the DocuSign API. What additional details do you need?

**Comment by Adrian-DS-Support** [comment]
Thank you Oscar, 

Our API reference includes a few methods for evidence:
https://docusign.github.io/docusign-esign-node-client/module-model_IdEvidenceViewLink.html

Our apologies if the process you need has not been developed yet. 

https://developers.docusign.com/docs/idevidence-api/reference/

Regards
Adrian
Developer Support

**Comment by garg-mudit** [comment]
Hi @OscarGodson,
Currently we don't provide any base SDK which provides you with the requested functionality.

 But, you can copy the ApiClient within this repo, along with any required supporting file so as to mimic similar functionality.
 
 Thank You.
